cucumber/api/features/step_definitions/authz_steps.rb

```diff
@@ -82,7 +82,7 @@
 end
 
 Given(/^I create a binary secret value?$/) do
-  @value = Random.new.bytes(16)
+  @value = Random.new.bytes(16).force_encoding(Encoding::UTF_8)
```

Comment:
Ah yes I was considering doing an array of bytes comparison previously. That works as well. @dividedmind I think UTF-8 is RestClient's default way of returning binary data. Maybe we can change the encoding, though.